scripts/deployment/l2/globals_gnosis_mainnet.json

```diff
@@ -24,7 +24,7 @@
   "multisigProxyHash130": "0xb89c1b3bdf2cf8827818646bce9a8f6e372885f8c55e5c07acbd307cb133b000",
   "stakingNativeTokenAddress":"",
   "stakingTokenAddress":"0xEa00be6690a871827fAfD705440D20dd75e67AB1",
-  "minStakingDepositLimit":"1000000000000000000000",
+  "minStakingDepositLimit":"10000000000000000000000",
```

Comment:
isn't technically 5k enough? since it works as 5k*2?

Reply:
Correct, 5k is enough, but let's double it just in case more is needed.